website/static/dist/devicescript-vm.js

```diff
@@ -327,7 +327,7 @@ var Exts;
     }
     Exts.devsClientDeploy = devsClientDeploy;
     /**
-     * Initalises the virtual machine data structure.
+     * Initialises the virtual machine data structure.
```

Comment:
My tooling wanted to skip this file (because it correctly guessed it was autogenerated). I've foolishly overridden its opinion and included fixes. If I'm wrong (quite likely), this change (or anything else) can be dropped.